CHANGELOG.md

```diff
@@ -1,3 +1,70 @@
+## 2023-05-16 v0.17.0
+
+## What's Changed
+
+Release highlights:
+* New rest api for Waku Store protocol
+* Removed Waku2 Swap protocol
+* New Filter protocol implentations
```

Comment:
```suggestion
* New Filter protocol implementation. See [12/WAKU2-FILTER](https://rfc.vac.dev/spec/12/).
```

In general I think it's a good idea to link people to documents where available.